designs/2019-core-options/README.md

```diff
@@ -0,0 +1,139 @@
+- Start Date: 2019-05-12
+- RFC PR: https://github.com/eslint/rfcs/pull/22
+- Authors: Toru Nagashima ([@mysticatea](https://github.com/mysticatea))
+
+# Configuring core options in Config Files
+
+## Summary
+
+This RFC adds several properties into config files to configure core options. People can use config files (including shareable configs!) instead of CLI options in order to configure some linter behavior.
+
+## Motivation
+
+We cannot configure some linter's behavior with config files, especially, shareable configs. It's convenient if we can configure those in shareable configs.
+
+## Detailed Design
+
+This RFC adds four properties to config files.
+
+```js
+// .eslintrc.js
+module.exports = {
+    disableInlineConfig: false,              // Corresponds to --no-inline-config
```

Comment:
I do think `disableInlineConfig` is a better property name than `noInlineConfig`, but I do think it's less of a burden on end users if we keep the names consistent.

Reply:
Good catch. I'm OK with `noInlineConfig`.